numbers/guides/hostedMessaging.md

```diff
@@ -427,6 +519,19 @@ Content-Type: application/xml; charset=utf-8
 </ImportTnOrder>
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->getImportTnOrder("order_id"));
```

Comment:
Same as above, break out to assign as variable

Reply:
done 👍